dove/tests/cmd_dove_clean.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg(test)]
+
+use std::path::{Path, PathBuf};
+use std::fs::{remove_dir_all, read_to_string};
+use fs_extra::file::write_all;
+use dove::cli::execute;
+
+/// $ cargo run -- clean
+/// $ dove clean
+/// project name: demoproject_16
+#[test]
+fn clean() {
+    // Path to dove folder
```

Comment:
Why don't you use `tempfile` crate to create and automatically remove temporary directories? There are several issues with the code:
1. It contains long, repeating code in every tests. It hides real purpose and lead to an enormous PR size
2. In case of unexpected panics in the test, `project_folder` won't be removed.